.github/hacktoberfest.md

```diff
@@ -0,0 +1,24 @@
+# Hacktoberfest
+
+During October 2022, Astro’s docs repo will accept your PRs for [Hacktoberfest](https://hacktoberfest.com/). Here’s how to take part!
+
+1. Make sure you’ve [registered with the Hacktoberfest organizers](https://hacktoberfest.com/).
+
+2. Open PRs to this repo between October 1 and October 31. (The earlier the better!)
+
+3. Let us know you’re taking part by requesting for your PR to be labelled as `hacktoberfest-accepted`.
```

Comment:
Is there a particular way to do this? Do they just write that in the PR somewhere, or uncomment out something? Should **we** add a label in Docs? (I think only we can actually label things though, right? Or, can they add a label themselves?)

We could probably decide, then make this a bit more explicit.

Reply:
I’ve created the label and it will be up to maintainers to apply it, so that’s why I said to “request” labelling. (Except for i18n review log PRs which I’ve configured the auto-labeller for.)